cross/chromaprint/Makefile

```diff
@@ -0,0 +1,43 @@
+PKG_NAME = chromaprint
+PKG_VERS = 1.2
+PKG_EXT = tar.gz
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://bitbucket.org/acoustid/chromaprint/downloads
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+
+DEPENDS = cross/ffmpeg-2.2.1
```

Comment:
Can you check if using `BUILD_DEPENDS` here works for ffmpeg (BUILD_DEPENDS skips PLIST processing)? 
If so, you don't have to clean up the stuff in the postinstall here: https://github.com/SynoCommunity/spksrc/pull/1622/files#diff-b18021950851bbb4e94acd6843c4156cR37.